packages/block-editor/src/components/use-moving-animation/index.js

```diff
@@ -82,13 +86,23 @@ function useMovingAnimation( {
 			}
 		}
 
-		if ( prefersReducedMotion ) {
+		if (
+			window.matchMedia( '(prefers-reduced-motion: reduce)' ).matches ||
+			isTyping() ||
+			getGlobalBlockCount() > BLOCK_ANIMATION_THRESHOLD
```

Comment:
I think this condition would be good as a const with a comment. (like why we're disabling the animation in these conditions)

Reply:
Will add one